test/unit/spec/nbextensions/appCell2/appCell-spec.js

```diff
@@ -0,0 +1,82 @@
+define([
+    '/narrative/nbextensions/appCell2/appCell',
+    'testUtil',
+    'narrativeMocks',
+    'json!/test/data/NarrativeTest.test_simple_inputs.spec.json',
+], (AppCell, TestUtil, Mocks, TestAppSpec) => {
+    'use strict';
+
+    fdescribe('test the base AppCell2 module', () => {
+        it('loads with expected functions', () => {
+            ['make', 'isAppCell'].forEach((fn) => {
+                expect(AppCell[fn]).toEqual(jasmine.any(Function));
+            });
+        });
+
+        describe('create AppCell tests', () => {
+            afterEach(() => {
+                TestUtil.clearRuntime();
+            });
+
+            it('makes an AppCell without starting it', () => {
+                const fakeCell = Mocks.buildMockCell('code');
+                const appCell = AppCell.make({ cell: fakeCell });
+                ['setupCell', 'upgradeToAppCell'].forEach((fn) => {
+                    expect(appCell[fn]).toEqual(jasmine.any(Function));
+                });
+            });
+
+            it('makes a new AppCell from scratch', () => {
+                const fakeCell = Mocks.buildMockCell('code');
+                const appCell = AppCell.make({ cell: fakeCell });
+                const appTag = 'release';
+                const appType = 'app';
+
+                // mock NMS.get_method_spec, NMS.get_method_full_info, Catalog.get_exec_aggr_stats
+                return appCell.upgradeToAppCell(TestAppSpec, appTag, appType);
```

Comment:
does this test anything?

Reply:
@ialarmedalien fleshed out the remaining tests in that module.